site/profile/manifests/cvmfs.pp

```diff
@@ -120,3 +122,31 @@
   selinux::boolean { 'use_fusefs_home_dirs': }
 
 }
+
+# Create a local cvmfs user
+class profile::cvmfs::local_user (
+  String $uname = 'cvmfs',
+  String $group = 'cvmfs-reserved',
+  Integer $uid = 13000004,
+  Integer $gid = 8000131,
+  String $selinux_user = 'unconfined_u',
+  String $mls_range = 's0-s0:c0.c1023',
+)
+{
+  group { $group:
+    ensure => present,
+    gid    => $gid,
+    before => User[$uname]
+    }
+
+  user { $uname:
+    ensure     => present,
+    forcelocal => true,
+    uid        => $uid,
+    gid        => $gid,
+    managehome => true,
```

Comment:
The cvmfs user does not require a home.
Set `managehome => false,`
Add `shell => '/usr/sbin/nologin',` to deny login as `cvmfs`.

Reply:
Configure home path:
```home => '/var/lib/cvmfs'``` 